### PR TITLE
Fix checkstyle issues

### PR DESCRIPTION
**Pull request by madurangasiriwardena**

### CODE REVIEW

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java [hunk 1]
```diff
@@ -97,8 +98,8 @@
 
         String loggedInUser = CarbonContext.getThreadLocalCarbonContext().getUsername();
 
-        if (log.isDebugEnabled()) {
-            log.debug("Adding a consumer secret for the logged in user " + loggedInUser);
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Adding a consumer secret for the logged in user " + loggedInUser);
```
Comment (tharindu-bandara) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java]: Shall we add a colon `:` before the user parameter?

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java [hunk 2]
```diff
@@ -887,52 +889,52 @@ public OAuthRevocationResponseDTO revokeAuthzForAppsByResourceOwner(
...
-                                            .revokeOAuthConsentByApplicationAndUser((
-                                                    (AuthenticatedUser) authzUser).getAuthenticatedSubjectIdentifier(),
-                                                                                    tenantDomain, appName);
+                                            .revokeOAuthConsentByApplicationAndUser(
+                                                    authzUser.getAuthenticatedSubjectIdentifier(),
+                                                    tenantDomain, appName);
                                 } catch (IdentityOAuth2Exception e) {
                                     String errorMsg = "Error occurred while removing OAuth Consent of Application " +
-                                                      appName + " of user " + userName;
+                                            appName + " of user " + userName;
```
Comment (tharindu-bandara) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java]: Shall we add a colon before the parameters?

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java [hunk 3]
```diff
@@ -998,16 +1000,16 @@ void triggerPostRevokeListeners(OAuthRevocationRequestDTO revokeRequestDTO,
...
         for (AccessTokenDO accessTokenDO : accessTokenDOs) {
             if (oAuthEventInterceptorProxy != null && oAuthEventInterceptorProxy.isEnabled()) {
                 try {
                     Map<String, Object> paramMap = new HashMap<String, Object>();
                     oAuthEventInterceptorProxy.onPostTokenRevocationByResourceOwner(revokeRequestDTO, revokeRespDTO,
-                                                                                    accessTokenDO, paramMap);
+                            accessTokenDO, paramMap);
                 } catch (IdentityOAuth2Exception e) {
-                    log.error("Error occurred with post revocation listener ", e);
+                    LOG.error("Error occurred with post revocation listener ", e);
```
Comment (tharindu-bandara) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java]: Space at the end of the log might not be required.

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java [hunk 4]
```diff
@@ -1137,36 +1140,36 @@ void validateTokenExpiryConfigurations(OAuthConsumerAppDTO oAuthConsumerAppDTO)
...
         }
     }
 
     void logOnInvalidConfig(String appName, String tokenType, long defaultValue) {
 
-        if (log.isDebugEnabled()) {
-            log.debug("Invalid expiry time value '0' set for " + tokenType + " in ServiceProvider: " + appName + ". "
-                      + "Defaulting to expiry value: " + defaultValue + " seconds.");
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Invalid expiry time value '0' set for " + tokenType + " in ServiceProvider: " + appName + ". "
```
Comment (tharindu-bandara) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java]: Shall we add a colon before the token type parameter?

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java [hunk 5]
```diff
@@ -122,8 +123,8 @@
 
         if (userName == null) {
             String msg = "User not logged in to get all registered OAuth Applications";
```
Comment (pamodaaw) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java]: Shall we add the full stop for the debug messages?

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java [hunk 6]
```diff
@@ -700,36 +701,36 @@ void updateAppAndRevokeTokensAndAuthzCodes(String consumerKey,
...
+            if (LOG.isDebugEnabled()) {
+                LOG.debug("Access tokens and token of users are removed from the cache for OAuth App with " +
+                        "consumerKey: " + consumerKey);
             }
 
             Set<String> authorizationCodes = OAuthTokenPersistenceFactory.getInstance().getAuthorizationCodeDAO()
-                                                                         .getActiveAuthorizationCodesByConsumerKey
-                                                                                 (consumerKey);
+                    .getActiveAuthorizationCodesByConsumerKey
+                            (consumerKey);
```
Comment (pamodaaw) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java]: Can't these two lines be combined?

**Pull request merged**